src/float/inv.rs

```diff
@@ -7,7 +7,29 @@ use super::F32;
 impl F32 {
     /// Fast approximation of `1/x`.
     pub fn inv(self) -> Self {
-        Self(f32::from_bits(0x7f00_0000 - self.0.to_bits()))
```

Comment:
It seems like `checked_sub` could potentially be leveraged here, e.g.

```rust
match 0x7f00_0000.checked_sub(self.0.to_bits()) {
```

Reply:
Very good point, I updated that. The remaining code remains the same, all the other cases are still error conditions of the subtraction.